### ISSUE TITLE
Better UDF visualization

### ISSUE THREAD

**Issue by wajda** [AbsaOSS/spline#29]
Thanks to Alper Topcuoglu
https://anchormen.nl/blog/big-data-services/spline/ (see Case 2 (Handling UDF + SQL))

Relates to #28

**Comment by mn-mikke** [comment]
Is it possible? There is not much we could highlight about UDFs except their names... Even Spark Catalyst has to treat them as a black box since information extraction from a piece of byte code is really tricky problem.

**Comment by wajda** [comment]
No, we're not talking about introspecting UDFs. The point of this ticket is to improve the way how calls to UDFs are represented on the UI. We now have those ugly toString() representations of expressions. Surely we could make them more readable.
